src/main/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/OverflowDbEdge.java

```diff
@@ -126,7 +135,8 @@ public Graph graph() {
 
   @Override
   public void remove() {
-    throw new RuntimeException("Not supported.");
+    outVertex.get().removeEdge(Direction.OUT, label(), inVertex, outBlockOffset);
```

Comment:
Add offset initalizations here if the are still unintialized. Use `initializeInFromOutOffset` respectively `initializedOutFromInOffset`.

Reply:
yup, the test that included serialization was failing before doing so. done now.